src/ui/connect-dashboard-ui/src/FamilyHubs.RequestForSupport.Web/Pages/La/Dashboard.cshtml.cs

```diff
@@ -1,17 +1,19 @@
+using FamilyHubs.Referral.Core.ApiClients;
 using FamilyHubs.ReferralService.Shared.Dto;
 using FamilyHubs.ReferralService.Shared.Enums;
 using FamilyHubs.ReferralService.Shared.Models;
-using FamilyHubs.RequestForSupport.Core.ApiClients;
 using FamilyHubs.RequestForSupport.Web.LaDashboard;
 using FamilyHubs.RequestForSupport.Web.Models;
 using FamilyHubs.RequestForSupport.Web.Pages.Shared;
 using FamilyHubs.RequestForSupport.Web.Security;
 using FamilyHubs.SharedKernel.Identity;
+using FamilyHubs.SharedKernel.Identity.Models;
 using FamilyHubs.SharedKernel.Razor.Dashboard;
 using FamilyHubs.SharedKernel.Razor.FamilyHubsUi.Options;
 using FamilyHubs.SharedKernel.Razor.Pagination;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.Extensions.Options;
+using IReferralClientService = FamilyHubs.RequestForSupport.Core.ApiClients.IReferralClientService;
```

Comment:
Is this included this way due to a clash or is it R# doing it?

Reply:
I think that was rider doing something odd. I've removed the references as they come from another directly